doc-content/enterprise-only/bpmn/bpmn-con.adoc

```diff
@@ -1,8 +1,63 @@
+--
+.General process properties
+[cols="30%,70%", options="header"]
+|===
+|Label
+|Description
 
+| *Name*
+| Enter the name of the process.
 
+| *Documentation*
+| Describes the process. The text in this field is included in the process documentation, if applicable.
+
+| *ID*
+| Enter an identifier for this process, such as `orderItems`.
+
+| *Package*
+| Enter the package location for this process in your {PRODUCT} project, such as `org.acme`.
+
+| *ProcessType*
+| Specify whether the process is public or private (or null, if not applicable).
+
+| *Version*
+| Enter the artifact version for the process.
+
+| *Ad hoc*
+| Select this option if this process is an ad hoc subprocess. (Currently not supported.)
+
+| *Process Instance Description*
+| Enter a description of the process purpose.
+
+| *Imports*
+| Click to open the *Imports* window and add any data object classes required for your process.
+
+| *Executable*
+| Select this option to make the process executable as part of your {PRODUCT} project.
+
+| *SLA Due Date*
+| Enter the date when the service level agreement (SLA) expires.
+
+| *Process Variables*
+a| Add any process variables for the process. Process variables are visible within the specific process instance. Process variables are initialized at process creation and destroyed on process completion. Variable *Tags* provide greater control over the variable behavior, such as whether the variable is `required` or `internal`. For more information about variable tags, see
+ifdef::PAM[]
+{URL_DESIGNING_BUSINESS_PROCESSES}#variables-con_designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
+endif::[]
+
+| *Metadata Attributes*
+| Add any custom metadata attribute name and value that you want to use for custom event listeners, such as a listener to implement some action when a metadata attribute is present.
+
+You can also use this field to configure role-based access to the process. To configure role-based access, set the attribute *Name* to `securityRoles` and set the attribute value to the relevant group or groups defined for the process, such as `employees,managers`.
+
+| *Global Variables*
+|  Add any global variables for the process. Global variables are visible to all process instances and assets in a project. Global variables are typically used by business rules and constraints, and are created dynamically by the rules or constraints.
+|===
+--
+
+A process is a container for a set of modeling elements. It contains elements that specify the execution workflow of a business process or its parts using flow objects and flows. Each process has its own BPMN2 diagram. {PRODUCT} contains the legacy process designer and the new process designer for creating BPMN2 diagrams. The new process designer has an improved layout and feature set and continues to be developed. Until all features of the legacy process designer are completely implemented in the new process designer, both designers are available in {CENTRAL} for you to use.
```

Comment:
New designer already achieved feature parity and used as default one. Legacy designer is used only to open old BPMN2 diagrams (with BPMN2 extension). All new diagrams by default creates in new designer and it is not possible to create new diagram in old designer. But it is possible to upload file with BPMN2 extension and it will be opened in old designer.

Reply:
thanks for the input. Changing the sentence as follows:

.... {PRODUCT} contains the new process designer for creating BPMN2 diagrams and the legacy process designer to open the old BPMN2 diagram that consists .bpmn2 extension. The new process designer has an improved layout and feature set and continues to develop. By default, the new diagrams are created in the new process designer. However, the old process designer is used to upload a file with .bpmn2 extension.

Reply:
Hi @hmanwani-rh, thank you for update, looks good, just one note: legacy designer not used to upload, but open already uploaded files with `.bpmn2` extension. So if you **upload** with `.bpmn` extension it will be opened in the new editor, if you **upload** process with `.bpmn2` extension (using BC upload asset capability) process will be opened in the legacy editor.

If  you **create** new BPMN diagram in Business Central, new asset with `.bpmn` extension will be created and always will be opened in the new editor.

Reply:
Thanks @hasys, Have removed "However, the old process designer is used to upload a file with .bpmn2 extension." sentence.